### ISSUE TITLE
[Bug]: Typo error BarLifeDrawPorition should be BarLifeDrawPosition

### ISSUE THREAD

**Issue by pertusatti22** [RafaelBarbosatec/bonfire#515]
### What happened?

Has been a while I want to develop some games, and bonfire helping me a lot with!
While watching the series of Tower Defense, I noticed this typo error and thinking it a good way to start my contribution. 

I clone the repo and try to figure it out and send a PR, and at your disposal to start contributing.
Thanks again for bonfire.

Regards!

### Steps to reproduce?

```Markdown
1. Create some attackable
2. setup LifeBar
3. barLifeDrawPosition

setupLifeBar(barLifeDrawPosition: BarLifeDrawPorition.bottom);
```

### What did you expect to happen?

setupLifeBar(barLifeDrawPosition: BarLifeDrawPosition.bottom);

### Bonfire version

3.8.7

**Comment by pertusatti22** [comment]
The fix was done, just don't have permission,
Waiting for contact
__
Commit message
fix: correct typo in BarLifeDrawPorition to BarLifeDrawPosition (#515)
## How
- Renamed BarLifeDrawPorition to BarLifeDrawPosition in the barlife_component.dart.
- Check for usages and reference errors in use_barlife.dart

___

Can't create a new pull request: Push failed: remote: Permission to RafaelBarbosatec/bonfire.git denied to pertusatti22. unable to access 'https://github.com/RafaelBarbosatec/bonfire.git/': The requested URL returned error: 403

**Comment by RafaelBarbosatec** [comment]
Thanks so much for this reporting!
Available in the version 3.9.0
